application/event_processor/dos.py

```diff
@@ -39,86 +43,127 @@ class DoSService:
     ]
 
     def __init__(self, db_cursor_row):
-
-        # Sets the attributes of this object to those found in the db row
+        """Sets the attributes of this object to those found in the db row
+        Args:
+            db_cursor_row (dict): Change Request changes
+        """
         for i, attribute_name in enumerate(self.db_columns):
-            attrbute_value = db_cursor_row[i]
-            setattr(self, attribute_name, attrbute_value)
+            attribute_value = db_cursor_row[i]
+            setattr(self, attribute_name, attribute_value)
 
-    def __repr__(self):
+    def __repr__(self) -> str:
+        """Returns a string representation of this object"""
         if self.publicname is not None:
             name = self.publicname
         elif self.name is not None:
             name = self.name
         else:
             name = "NO-VALID-NAME"
 
-        return f"<uid={self.uid} ods={self.odscode} type={self.typeid} " f"status={self.statusid} name='{name}'>"
+        return f"<uid={self.uid} ods={self.odscode} type={self.typeid} status={self.statusid} name='{name}'>"
 
-    def ods5(self):
-        """First 5 digits of odscode"""
-        return self.odscode[0:5]
-
-    def get_changes(self, nhs_entity) -> dict:
+    def get_changes(self, nhs_entity: NHSEntity) -> dict:
         """Returns a dict of the changes that are required to get
         the service inline with the given nhs_entity
         """
-
         changes = {}
+        add_field_to_change_request_if_not_equal(changes, "website", self.web, nhs_entity.Website)
```

Comment:
Move all change keys to a constants file so here and in unit tests we can avoid duplicating these strings, and if any changes to the keys accepted by DoS we can change them in one place 


`add_field_to_change_request_if_not_equal(changes, DoSChangeKyes.POSTCODE, self.postcode, nhs_entity.Postcode)`

Reply:
Updated,I added this to the change_request.py